### PR TITLE
Adds RequestedDocumentsLater to list of required document types

### PR DESCRIPTION
**Pull request by jonsgreen**
- (#172660949) Require at least one requested document

### CODE REVIEW

**Comment by jonsgreen** [comment]
Note: using the shared view code for required documents changes the text of the button/links to just be "Continue". Hopefully that makes sense enough in this context but otherwise we will need to make some changes to accommodate different text.

File: app/lib/bank_details_pdf.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'mini_magick'
+
```
Comment (bengolder) [inline app/lib/bank_details_pdf.rb:2]: This looks unrelated, why was it included?

Comment (jonsgreen) [reply app/lib/bank_details_pdf.rb:2]: Yes, it is unrelated. My test failed so I think it must be a failure that only happens when run in a certain order. Generally the suite passes. I probably should remove it and maybe we should research this separately. I am curious if that spec fails for you if you run it individually .

Comment (bengolder) [reply app/lib/bank_details_pdf.rb:2]: Yes tests are passing for me
